docs/modules/ROOT/pages/notitle.adoc

```diff
@@ -2,14 +2,14 @@
 :description: The notitle option hides a section title in the document body. It can also add a preamble, preface, or imported PDF entry to the TOC.
 
 Asciidoctor PDF adds support for the `notitle` option on a section as a way to express that you want the title of that section to be hidden in the document body.
-This option can also be used to add a TOC entnry for a preamble, anonymous preface, or imported PDF page.
+This option can also be used to add a TOC entry for a preamble, anonymous preface, or imported PDF page.
 
 == notitle option
 
-When the `notitle` option is set on a section, the converter will hide the section title in the body of the document, but still display it in the TOC and in cross references to that section.
+When the `notitle` option is set on a section, the converter will hide the section title in the body of the document, but still display it in the TOC and in cross-references to that section.
```

Comment:
Let's keep "cross-references" as two words. Hyphenations usually fall out of favor pretty quickly once a term is established.